Embassy/HTTP/DefaultHTTPServer.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  DefaultHTTPServer.swift
+//  Embassy
+//
+//  Created by Fang-Pen Lin on 5/19/16.
+//  Copyright © 2016 Fang-Pen Lin. All rights reserved.
+//
+
+import Foundation
+
+public final class DefaultHTTPServer: HTTPServer {
```

Comment:
hmmm, in terms of naming, maybe it would be better to be `SimpleHTTPServer` or `BasicHTTPServer`?